.npmignore

```diff
@@ -2,3 +2,10 @@ test
 examples
 out
 Makefile
+package-lock.json
```

Comment:
i believe npm always ignores this by default (could look up) but might as well make explicit.